cmd/juju/controller/register.go

```diff
@@ -233,74 +348,81 @@ func (c *registerCommand) maybeSetCurrentModel(ctx *cmd.Context, store jujuclien
 		if err != nil {
 			return errors.Trace(err)
 		}
-		fmt.Fprintf(ctx.Stderr, "\nCurrent model set to %q.\n\n", modelName)
-	} else {
-		fmt.Fprintf(ctx.Stderr, `
+		fmt.Fprintf(ctx.Stderr, "\nCurrent model set to %s.\n", modelName)
```

Comment:
Please revert, so we are consistent with use of %q for controller names in sentences.